cmdb.js

```diff
@@ -50,18 +51,64 @@ cmdb.prototype._fetch = function _fetch(locals, path, method, body) {
 }
 
 /**
+ * Helper function for requested count of pages and itemsfrom CMDB API
+ * @param {Object} [locals] - The res.locals value from a request in express
+ * @param {string} url - The url of the request to make
+ * @returns {Promise<Object>} The count of pages and items from CMDB (JSON-decoded)
+ */
+cmdb.prototype._fetchCount = function fetchCount(locals, url, timeout = 6000) {
```

Comment:
The start of this function is basically the same logic as `_fetch`.  Can't you just call `_fetch` here and then do the count logic on its response?